lua/starfall/libs_cl/bass.lua

```diff
@@ -231,6 +231,7 @@ end
 -- @return boolean Boolean of whether the bass is valid.
 function bass_methods:isValid()
 	local uw = unwrap(self)
+	if !uw then return false end
```

Comment:
unwrap shouldn't return nil. https://github.com/thegrb93/StarfallEx/blob/master/lua/starfall/instance.lua#L252

Reply:
Remove line 234 and change the return to `return uw.IsValid and uw:IsValid()`